x-pack/test/scalability/config.ts

```diff
@@ -63,10 +63,20 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
 
     kbnTestServer: {
       ...baseConfig.kbnTestServer,
+      serverArgs: [
+        ...baseConfig.kbnTestServer.serverArgs,
+        `--logging.loggers=${JSON.stringify([
+          ...getKibanaCliLoggers(baseConfig.kbnTestServer.serverArgs),
+          // Enable logger for the Ops Metrics
+          {
+            name: 'metrics.ops',
+            level: 'all',
+            appenders: ['default'],
```

Comment:
Super-nit: AFAIK, the `appenders` is not strictly needed (it will use the appenders set at root level)